animation_nodes/utils/bvh.pyx

```diff
@@ -0,0 +1,29 @@
+from random import random
+from mathutils import Vector
+
+cdef float hitsPerLocation(bvhTree, vector):
```

Comment:
Doesn't this return a boolean that says if the input is inside the volume? The function should be renamed accordingly.